odxtools/scaleconstr.py

```diff
@@ -25,8 +25,8 @@ class ScaleConstr:
 
     short_label: Optional[str]
     description: Optional[Description]
-    lower_limit: Optional[Limit]
-    upper_limit: Optional[Limit]
+    lower_limit: Limit
+    upper_limit: Limit
```

Comment:
Sure?

Reply:
yes: 
![Screenshot_20250212_181809](https://github.com/user-attachments/assets/2c940129-891b-4783-a738-23d1f40086f3)

(that's for ODX 2.0. 2.2 is the same)